src/components/card/services/addToCart.service.js

```diff
@@ -12,11 +13,13 @@ const addToCartService = async ({ url, body, token, successStatus, cartDispatch,
                 payload: { cart: response.data.cart },
             });
             setCtaBtnText("Go To Cart");
+            toast.success(body.product.title.slice(0, 13).trim() + "... Added to Cart");
         } else {
             throw new Error(response.status, "<-- error code");
         }
     } catch (error) {
-        alert(error);
+        toast.error("Error Occured! Please Try Again.");
+        console.error(error)
```

Comment:
you can remove consoles.

Reply:
This here is not a `console.log`, it's `console.error()` method which is used to display error in  the console.